src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -738,6 +740,10 @@ def setUp(self):
                 self.access_points, self.access_points_suffix)
         self.host_info.access_points = self.access_points
 
+        # Toggle whether to dump server ULT stacks on failure
+        self.__dump_engine_ult_on_failure = self.params.get(
+            "dump_engine_ult_on_failure", "/run/*", True)
```

Comment:
Many of the other params consumed by these Test classes - like whether to start agents/servers once - are specified under `setup` in the test yaml.  This seems like one of those options:
```suggestion
            "dump_engine_ult_on_failure", "/run/setup/*", True)
```

FYI, there does seem to be one outlier to this: `self.skip_add_log_msg`